src/components/InAllLanguages.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+	<div class="wikibase-termbox wikibase-termbox--inAllLanguages">
+		<div class="wikibase-termbox__switch">
+			<a :href="link" @click.prevent="showAllLanguagesSwitch()">{{ message( MESSAGE_KEYS.ALL_LANGUAGES ) }}</a>
+		</div>
+		<div v-if="showAllLanguages">
+			<div>
+				<div v-for="language in allLanguagesLanguageKeys">
+					<Dab :modifier="modifier" :language="language" />
+				</div>
+			</div>
+			<div class="wikibase-termbox__actions">
```

Comment:
This actions element has to go. Editing for terms that are not the user language is only going to be introduced in a much later story.